### PR TITLE
Update package.json to add build script

### LINKED ISSUE #12: [apps] Add "build" script to React and Svelte apps
The React and Svelte projects should have a script to build de app.

### PR DESCRIPTION
**Pull request by JavierOrtunoRoig**
This PR fix #12.
I have added a script to build the Svelte and React applications

### CODE REVIEW

File: apps/react-app/package.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "vite --port 8080 --open"
+    "start": "vite --port 8080 --open",
+    "build": "vite build"
```
Comment (Nasdan) [inline apps/react-app/package.json:7]: Why do you need the build command on these apps?

Comment (Nasdan) [reply apps/react-app/package.json:7]: After talk with @brauliodiez, these apps will be deployed in a web server so we alse will need to build them. Thanks

Comment (Nasdan) [reply apps/react-app/package.json:7]: Maybe for publish packages, I think we should update the `publish-packages` command to: `"publish-packages": "turbo run build --filter=./packages/* && changeset publish"`

Comment (JavierOrtunoRoig) [reply apps/react-app/package.json:7]: I have changed the publish-packages command as you suggested

**Pull request merged**